src/eventer/eventer_impl.c

```diff
@@ -61,6 +63,22 @@ static uint32_t init_called = 0;
 #define NS_PER_MS 1000000
 #define NS_PER_US 1000
 
+static unsigned long __ck_hash_from_uint64(const void *key, unsigned long seed) {
+  return (*(uint64_t *)key) ^ seed;
+}
+static bool __ck_hash_compare_uint64(const void *a, const void *b) {
+  return *(uint64_t *)a == *(uint64_t *)b;
+}
+static void * gen_malloc(size_t r) { return malloc(r); }
+
+static void gen_free(void *p, size_t b, bool r) {
+  (void)b; (void)r; free(p); return;
+}
+static struct ck_malloc malloc_ck_hs = {
+  .malloc = gen_malloc,
+  .free = gen_free
```

Comment:
surprised you didn't have to implement `.realloc`.  I think ubuntu compiler might complain if you don't

Reply:
This is how we do it in the jobq system and it works (or appears to)

Reply:
Probably just the C++ compiler that would complain